design/one-pager-package-format-v2.md

```diff
@@ -256,7 +255,7 @@ that will, when invoked:
 
 1. Emit the content of `crossplane.yaml` to stdout.
```

Comment:
nitpick: this still somewhat indicates that the `crossplane.yaml` will be emitted first

Reply:
Switched to an unordered list.